userguide/one_shot_object_detection/README.md

```diff
@@ -0,0 +1,109 @@
+# One-Shot Object Detection
+
+One-Shot object detection is the task of learning to detect a set of categories from a handful of images. Unlike the Object Detector which requires many varied examples of objects in the real world, the One-Shot detector only requires a single canonical example of an object to train the classifier on and performs predictions for that category in the wild.
+
```

Comment:
I think it might be good to manage expectations and give some context: OSOD can work quite well for things like traffic signs or logos, but it's probably not going to do very well for detecting cats or dogs. I think it might be good to mention OSOD is really only intended for cases where all instances of a category really look the same (at least look the same to us humans).

Reply:
Does the edit look good?

Reply:
Rather than just giving examples of things which are good and not good for OSOD, I think it would be good to explain the reason. Some thing like: OSOD works best when there isn't much expected variance in the examples.